source/_integrations/reolink.markdown

```diff
@@ -18,6 +18,8 @@ ha_dhcp: true
 
 The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
 
+If you appreciate the reolink integration and want to support its development, please consider becoming a [![](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=GitHub&color=%23fe8e86)](https://github.com/sponsors/starkillerOG) of the upstream library.
```

Comment:
I'm not sure if we want to do this, as it will set a precedence to riddle our documentation with requests for sponsor links for a lot of things.

Will check.

../Frenck

Reply:
Thanks for checking @frenck

Maybe a more clean (and uniform for other integrations) sollution would be to put it in the sidebar.
But I guess that requires a architecture discusion or something?
And I would have no idea how to actually get it in the sidebar ;)

![afbeelding](https://user-images.githubusercontent.com/14811189/217938837-1dda712d-222a-45ba-a5d8-56f76d670e2e.png)

Reply:
@frenck did you check up on this?

Reply:
I do not have an answer yet, this isn’t forgotten.

Reply:
Would it be possible to get a decision before HA 2023.4 gets released?
I expect quite some attention for the reolink integration upon release of 2023.4 since it adds lots of new features.

Reply:
@frenck it's been 2 months, has some decision already been made?
I recently also got a affiliate link that I added to this PR.

Would mean a lot for me to get this merged.